deploy/FullDeploy.js

```diff
@@ -103,6 +103,13 @@ module.exports = async function (hre) {
         contract: "OracleAdapter",
     })
 
+    const gasOracleAdapter = await deploy("GasOracleAdapter", {
+        from: deployer,
+        log: true,
+        args: [gasOracle.address],
+        contract: "OracleAdapter",
+    })
+
```

Comment:
Should this be being used in the deployment?

Reply:
ah I was going to add it in but decided that was most likely a separate PR as there is some logic to go into the perp itself. 

I'll remove this for now and add it in another PR